src/v/cloud_storage/remote_partition.cc

```diff
@@ -936,6 +935,10 @@ remote_partition::aborted_transactions(offset_range offsets) {
         meta_to_materialize.clear();
         auto cur_res = co_await _manifest_view->get_cursor(offsets.begin);
         if (cur_res.has_failure()) {
+            if (cur_res.error() == error_outcome::shutting_down) {
+                throw std::runtime_error("Async manifest view shutting down");
+            }
+
```

Comment:
Can you remind me how this gets propagated to clients? Is this caught somewhere and passed off as a retriable Kafka error code?

Reply:
Good question. Went spelunking for this. We will terminate the connection by throwing a `runtime_error` from here. This was would happen previously too, so if we wish to change that we could do it in separate pr.